pkg/scheduler/framework/runtime/registry_test.go

```diff
@@ -14,13 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package runtime
 
 import (
 	"reflect"
 	"testing"
 
 	"k8s.io/apimachinery/pkg/runtime"
+
```

Comment:
No need for space and also rename variables instead of the import

Reply:
these spaces are being created by `gofmt`, it creates spaces to keep different packages separate like `k8s.io/apimachinery/*`, `"k8s.io/kubernetes/*` etc. I've manually removed them. Pls let me know if theres some other command you guys use to do linting.

Reply:
These spaces are being created by `gofmt`, it creates spaces to keep different types of packages separate like `k8s.io/apimachinery/*`, `k8s.io/kubernetes/*` etc. I've manually removed them. Pls let me know if theres some other command you guys use to do linting.